app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
-    <string name="app_name">ParkingSystemJava</string>
-</resources>
\ No newline at end of file
+    <string name="app_name">estacionAR</string>
+    <string name="main_activity_textview_tittle">EstacionAR</string>
```

Comment:
same here this should be "main_activity_text_view_title"

Reply:
Done